src/utils/integration_utils.js

```diff
@@ -54,6 +54,14 @@ export function checkDialogElementForError(elem: DialogElement, value: Object):
                 };
             }
         }
+    } else if (type === 'radio') {
+        const options = elem.options;
+        if (value && Array.isArray(options) && !options.some((e) => e.value === value)) {
```

Comment:
I see a case that might be problematic:
Value is falsy and it isn't on the list of options. 
I'd go for a explicit check for undefined